src/main/java/org/corfudb/cmdlets/ICmdlet.java

```diff
@@ -58,6 +58,12 @@ default CorfuRuntime configureRuntime(Map<String, Object> opts) {
                 .connect();
     }
 
+    default CorfuRuntime configureRuntimeAddrPort(Map<String, Object> opts) {
```

Comment:
What does configureRuntimeAddrPort do? It needs either a self explanatory method name or at least a comment describing what it does.